.github/workflows/dotnet-core-win.yml

```diff
@@ -0,0 +1,48 @@
+name: .NET Windows
+
+on:
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build-test:
+    runs-on: windows-latest
+
+    strategy:
+      matrix:
+        version: [net452,net46,net461,netcoreapp3.1]
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Setup .NET Core
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: 3.1.x
+        
+    - name: Install dependencies
+      run: dotnet restore
+
+    - name: Build
+      run: dotnet build --configuration Release --no-restore
+
+    - name: test ${{ matrix.version }}
+      run: dotnet test **\bin\**\${{ matrix.version }}\*Tests.dll
+    # - uses: actions/cache@v2
```

Comment:
please add comment why these are commented out.

Reply:
hi @cijothomas , i will remove that in my next commit